strings/textboxes.yml

```diff
@@ -651,62 +651,62 @@ KeyGK:
   Bombos Tablet: "A small key\nto the Kingdom"
   Ether Tablet: "A small key\nto the Kingdom"
 Crystal1:
-  Master Sword Pedestal: "Don't waste\nyour time!"
-  Bombos Tablet: "Don't waste\nyour time!"
-  Ether Tablet: "Don't waste\nyour time!"
+  Master Sword Pedestal: "A descendant\nof the Seven\nSages"
+  Bombos Tablet: "A descendant\nof the Seven\nSages"
+  Ether Tablet: "A descendant\nof the Seven\nSages"
 Crystal2:
-  Master Sword Pedestal: "Don't waste\nyour time!"
-  Bombos Tablet: "Don't waste\nyour time!"
-  Ether Tablet: "Don't waste\nyour time!"
+  Master Sword Pedestal: "A descendant\nof the Seven\nSages"
+  Bombos Tablet: "A descendant\nof the Seven\nSages"
+  Ether Tablet: "A descendant\nof the Seven\nSages"
 Crystal3:
-  Master Sword Pedestal: "Don't waste\nyour time!"
-  Bombos Tablet: "Don't waste\nyour time!"
-  Ether Tablet: "Don't waste\nyour time!"
+  Master Sword Pedestal: "A descendant\nof the Seven\nSages"
+  Bombos Tablet: "A descendant\nof the Seven\nSages"
+  Ether Tablet: "A descendant\nof the Seven\nSages"
 Crystal4:
-  Master Sword Pedestal: "Don't waste\nyour time!"
-  Bombos Tablet: "Don't waste\nyour time!"
-  Ether Tablet: "Don't waste\nyour time!"
+  Master Sword Pedestal: "A descendant\nof the Seven\nSages"
+  Bombos Tablet: "A descendant\nof the Seven\nSages"
+  Ether Tablet: "A descendant\nof the Seven\nSages"
 Crystal5:
-  Master Sword Pedestal: "Don't waste\nyour time!"
-  Bombos Tablet: "Don't waste\nyour time!"
-  Ether Tablet: "Don't waste\nyour time!"
+  Master Sword Pedestal: "A descendant\nof the Seven\nSages"
+  Bombos Tablet: "A descendant\nof the Seven\nSages"
+  Ether Tablet: "A descendant\nof the Seven\nSages"
 Crystal6:
-  Master Sword Pedestal: "Don't waste\nyour time!"
-  Bombos Tablet: "Don't waste\nyour time!"
-  Ether Tablet: "Don't waste\nyour time!"
+  Master Sword Pedestal: "A descendant\nof the Seven\nSages"
+  Bombos Tablet: "A descendant\nof the Seven\nSages"
+  Ether Tablet: "A descendant\nof the Seven\nSages"
 Crystal7:
-  Master Sword Pedestal: "Don't waste\nyour time!"
-  Bombos Tablet: "Don't waste\nyour time!"
-  Ether Tablet: "Don't waste\nyour time!"
+  Master Sword Pedestal: "A descendant\nof the Seven\nSages"
+  Bombos Tablet: "A descendant\nof the Seven\nSages"
+  Ether Tablet: "A descendant\nof the Seven\nSages"
 RescueZelda:
-  Master Sword Pedestal: "Don't waste\nyour time!"
-  Bombos Tablet: "Don't waste\nyour time!"
-  Ether Tablet: "Don't waste\nyour time!"
+  Master Sword Pedestal: " The royal\n treatment"
+  Bombos Tablet: " The royal\n treatment"
+  Ether Tablet: " The royal\n treatment"
```

Comment:
This one and the ganon text are the only ones that have leading spaces in there; is this intentional (and does it even make sense with the text engine)?

Reply:
It is intentional, since to my knowledge the pedestal and tablets don't auto-center like the Triforce.  If that's changed, though, those should be removed and some others should be adjusted.